CDCgov/prime-reportstream
# Fixes the "Prepare Deployment Branch" workflow.

This PR fixes the "Prepare Deployment Branch" workflow.


Test Steps:
1. Wait for the workflow to trigger on schedule and check if PR has been created

## Changes
- fixes the "Prepare Deployment Branch" workflow

## Checklist

### Testing

## Linked Issues
- Fixes #13463

## To Be Done
- @JosiahSiegel to review and merge a PR in his personal repo: https://github.com/JosiahSiegel/reliable-pull-request-action/pull/14
- Workflow updated to use his Action instead of the forked one
